src/main/java/internet/market/controllers/LoginController.java

```diff
@@ -0,0 +1,42 @@
+package internet.market.controllers;
+
+import internet.market.exceptions.AuthenticationException;
+import internet.market.lib.Injector;
+import internet.market.model.User;
+import internet.market.security.AuthenticationService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+
+public class LoginController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("internet.market");
+    private AuthenticationService authenticationService = (AuthenticationService) injector
+            .getInstance(AuthenticationService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/login.jsp").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String login = req.getParameter("login");
+        String password = req.getParameter("password");
+        try {
+            User user = authenticationService.login(login, password);
+            HttpSession session = req.getSession();
+            session.setAttribute("user_Id", user.getId());
```

Comment:
```suggestion
            session.setAttribute(USER_ID, user.getId());
```
let's make it constant as in filter

Reply:
done